timrlink.net.CLI/Actions/ProjectTimeXLSXImportAction.cs

```diff
@@ -94,21 +94,21 @@ protected override IEnumerable<Core.API.ProjectTime> ParseFile()
                     var duration = (long) (DateTime.FromOADate(double.Parse(columns[columnMapping.Duration].CellValue.Text)) - DateTime.FromOADate(0)).TotalMinutes;
                     var billable = columns[columnMapping.Billable].CellValue.Text == "1";
                     var changed = columns[columnMapping.ManuallyChanged].CellValue.Text == "1";
-                    var @break = (int) decimal.Parse(document.WorkbookPart.GetStringValue(columns[columnMapping.Break]));
+                    var @break = (int) decimal.Parse(columns[columnMapping.Break].CellValue.Text);
```

Comment:
what was the issue here?

Reply:
Got an exception after saving the excel. Was during development of the other features. I'll check back.

Reply:
A good question, at least why didn't we use `GetStringValue(WorkbookPart workbookPart, string column, IDictionary<string, Cell> cells)`. Did you try it with break times other than `0`?